quartz/components/Comments.tsx

```diff
@@ -46,7 +54,10 @@ export default ((opts: Options) => {
     )
   }
 
-  Comments.afterDOMLoaded = script
+  // make sure we actually need to load the script
+  if (Comments != null) {
+    Comments.afterDOMLoaded = script
+  }
```

Comment:
Then you don't have to do this check

Reply:
Done, that makes way more sense, thank you!